bzt/modules/blazemeter.py

```diff
@@ -1372,6 +1372,7 @@ def get_test_status_text(self):
 
 class MasterProvisioning(Provisioning):
     def get_rfiles(self):
+        jmeter_var_pattern = re.compile("\${.+\}")
```

Comment:
Maybe definition of it as constant of JMeterExecuter looks better as we have a lot places to reuse it (at least follow test)